internal/impl/kafka/enterprise/integration_test.go

```diff
@@ -40,14 +32,13 @@ import (
 
 	"github.com/redpanda-data/connect/v4/internal/impl/kafka"
 	"github.com/redpanda-data/connect/v4/internal/impl/kafka/enterprise"
+	"github.com/redpanda-data/connect/v4/internal/impl/kafka/redpandatest"
 	"github.com/redpanda-data/connect/v4/internal/license"
 	"github.com/redpanda-data/connect/v4/internal/protoconnect"
 	_ "github.com/redpanda-data/connect/v4/public/components/confluent"
 )
 
-func createKafkaTopic(ctx context.Context, address, id string, partitions int32) error {
-	topicName := fmt.Sprintf("topic-%v", id)
```

Comment:
I'm not sure why we had it like this, but it seems that the topic names are injected directly